packages/contracts-periphery/contracts/universal/op-nft/AttestationStation.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {
+    OwnableUpgradeable
+} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { Semver } from "@eth-optimism/contracts-bedrock/contracts/universal/Semver.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+contract AttestationStation is Initializable, OwnableUpgradeable, Semver {
```

Comment:
Why exactly is this ownable? I don't see `onlyOwner` used anywhere

Reply:
To be honest this doesn't need an `initialize` function because there are no side effects in the constructor, it should just work behind a proxy

Reply:
Yep doesn't need to be ownable

Reply:
Doesn't it need to be ownable so it can transfer ownership (which is onlyOwner)?  https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/master/contracts/access/OwnableUpgradeable.sol#L74

Reply:
There are no methods on the contract that require special access controls